docs/release_notes/8.0.0-1.md

```diff
@@ -0,0 +1,83 @@
+# Percona Server for MongoDB 8.0.1-1 ({{date.8_0_1}}) 
+
+[Installation](../install/index.md){.md-button}
+[Upgrade](../install/upgrade-from-70.md){.md-button}
+
+We are pleased to announce the release candidate of the new major version of Percona Server for MongoDB 8.0.1-1 – the drop in replacement for MongoDB Community Edition. Percona Server for MongoDB supports MongoDB Community protocols and drivers and enhances it with the [enterprise-grade features for free](../comparison.md), enabling you to effectively meet the requirements of your organization in terms of high-availability, reliability and data security. 
+
+This release of Percona Server for MongoDB includes the improvements and bug fixes of [MongoDB 8.0.0 Community Edition](https://www.mongodb.com/docs/manual/release-notes/8.0/#8.0.0---oct-2--2024) and [MongoDB 8.0.1 Community Edition](https://www.mongodb.com/docs/manual/release-notes/8.0/#8.0.1---upcoming)
```

Comment:
8.0.1 link was updated to: https://www.mongodb.com/docs/manual/release-notes/8.0/#8.0.1---oct-9--2024